packages/conventional-commit-lint/src/conventional-commit-lint.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2019 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Application } from 'probot';
+import lint from '@commitlint/lint';
+import { rules } from '@commitlint/config-conventional';
+import { PullsListCommitsResponseItem, Response } from '@octokit/rest';
+
+type Conclusion =
+  | 'success'
+  | 'failure'
+  | 'neutral'
+  | 'cancelled'
+  | 'timed_out'
+  | 'action_required'
+  | undefined;
+
+export = (app: Application) => {
+  app.on('pull_request', async context => {
+    // Fetch last 100 commits stored on a specific PR.
+    const commitParams = context.repo({
+      pull_number: context.payload.pull_request.number,
+      per_page: 100,
+    });
+    // Response object has a typed response.data, which has definitions that
+    // can be found here: https://unpkg.com/@octokit/rest@16.28.3/index.d.ts
+    let commitsResponse: Response<PullsListCommitsResponseItem[]>;
+    try {
+      commitsResponse = await context.github.pulls.listCommits(commitParams);
+    } catch (err) {
+      console.info(err);
```

Comment:
I find it interesting that you both write to the app log and write to the console.  What exactly is happening with the logging story here?

Reply:
whoops, just a log line left in by mistake.